lib/classroom_api.rb

```diff
@@ -34,12 +136,62 @@ def get_classroom_characteristics
     request["accept"] = 'application/json'
 
     response = http.request(request)
-    return JSON.parse(response.read_body)
+    response_json = JSON.parse(response.read_body)
+    if response_json['httpCode'].present?
+      @result['error'] = response_json['httpMessage'] + ". " + response_json['moreInformation']
+    else
+      @result['success'] = true
+      @result['data'] = response_json['Classrooms']['Classroom']
+    end
+    return @result
+    
+  end
+
+  def update_all_classroom_characteristics
+
+    classrooms = Room.where(rmtyp_description: "Classroom").where.not(facility_code_heprod: nil)
+    counter = 0
+    classrooms.each do |room|
+      if counter < 190 
```

Comment:
descriptive variable names help the readability of the code. Perhaps use **api_limiter_prevention_counter** would be better.